app/services/repository_rows/my_module_unassigning_service.rb

```diff
@@ -44,16 +44,19 @@ def succeed?
     def unassign_repository_rows_from_my_module(my_module)
       unassigned_names = my_module.my_module_repository_rows
                                   .joins(:repository_row)
-                                  .where(repository_rows: { repository: @repository, id: params[:selected_rows] })
-                                  .select('*, repository_rows.name AS name')
+                                  .where(repository_row_id: params[:rows_to_unassign])
+                                  .where(repository_rows: { repository: @repository })
+                                  .select('my_module_id,
+                                           my_module_repository_rows.id,
+                                           repository_rows.name AS name')
```

Comment:
What is happening here?

Reply:
Before it selected not correct id for unassign. You can check on feature branch.

Reply:
It is needed only change select to .select('my_module_repository_rows.*, repository_rows.name AS name')